DEFAULT_CONFIG.json5

```diff
@@ -337,8 +330,39 @@
         known_keys_file: null,
       },
     },
+    // acl: {
+    //   "enabled": false, //[true/false] acl will be activated only if this is set to true
```

Comment:
Use the same comment style as used in all the rest of the config file. I.e., above the parameter and starting with `///`.